src/CLMWrapper.cpp

```diff
@@ -208,7 +208,7 @@ void ClmWrapper::callback(const sensor_msgs::ImageConstPtr& msgIn)
         return;
     }
 
-    cv::cvtColor(cv_ptr->image.clone(), captured_image, CV_BGR2RGB);;
+    cv::cvtColor(cv_ptr->image.clone(), captured_image, CV_BGR2RGB);
```

Comment:
after this line you should not use `cv_ptr->image` any more, but directly captured_image. See below